emperor/support_files/js/controller.js

```diff
@@ -253,10 +254,42 @@ define([
                                  scope.resize(scope.$divId.width(),
                                               scope.$divId.height());
                                }});
+
+    // Set up the context menu
+    this.$contextMenu = $.contextMenu({
+      selector: '.emperor-plot-wrapper',
```

Comment:
Is there any way we can have this be the `this.$plotSpace` element, and not all elements with this class?